### PR TITLE
First steps for issues #53

### PR DESCRIPTION
**Pull request by shapkarin**
First steps for issues #53

Changes: count props, add default config, remove filename postfix

### CODE REVIEW

File: bin/initialize.js [hunk 1]
```diff
@@ -190,48 +190,21 @@ program
...
 							validate: function(input) {
 								let propNames = input.split(" ");
-								if (
-									propNames.length !==
-									Number(numberOfPropTypes)
-								) {
-									return `enter ${numberOfPropTypes} prop names`;
-								}
-								let regex = /^[a-zA-Z]+$/;
+								numberOfPropTypes = propNames.length;
```
Comment (shapkarin) [inline bin/initialize.js]: I do not like that it's in validate but don't know API

Comment (shapkarin) [reply bin/initialize.js]: oh i found https://github.com/SBoudrias/Inquirer.js

**Comment by rajikaimal** [comment]
I will test soon and merge this. Thanks! 🎉

**Comment by shapkarin** [comment]
I run that like:
```
node ./bin/initialize.js generate component Test That
```

But if run tests, "should create a parent react component" is failed.

**Comment by shapkarin** [comment]
Hope I already fix "generate" test  🙂

**Comment by shapkarin** [comment]
fixed "generate" test 🙂

**Comment by shapkarin** [comment]
I don't like that I save empty folder (_somebody can think that source there..._), but can't generate to the different directories. I also plan to do that at issues #53

**Comment by shapkarin** [comment]
I rename to test_folder but still think that all about tests must be in one folder but I found some other at "templates/src"

**Comment by shapkarin** [comment]
Anyway seems that tests need to move outside of "templates"

**Comment by shapkarin** [comment]
@rajikaimal I change test and .reactclirc to generate there and then remove generated folder. But what is the "templates/src" folder?

**Comment by shapkarin** [comment]
@rajikaimal what is the "templates/src" folder? I change test and .reactclirc to generate there and then remove generated folder.

**Comment by rajikaimal** [comment]
`templates` folder contains the default react template.

**Comment by shapkarin** [comment]
> `templates` folder contains the default react template.

@rajikaimal but what for src/components inside?

**Comment by shapkarin** [comment]
I changed test so it's generate to the "./test_folder/components" folder and then removes it.

**Comment by rajikaimal** [comment]
> but what for src/components inside?

We don't have an `src` folder. 

Other changes LGTM.

**Comment by rajikaimal** [comment]
Let's create issues for the other features/patches.

Thanks! 🙇 @shapkarin

**Pull request merged**